core/src/main/scala/org/apache/spark/deploy/JsonProtocol.scala

```diff
@@ -208,24 +208,63 @@ private[deploy] object JsonProtocol {
    *         `completeddrivers` a list of Json objects of [[DriverInfo]] of the completed drivers
    *         of the master
    *         `status` status of the master,
-   *         see [[org.apache.spark.deploy.master.RecoveryState.MasterState]]
+   *         see [[org.apache.spark.deploy.master.RecoveryState.MasterState]].
+   *         If `field` is not None, the Json object will contain the matched field.
```

Comment:
`is not None` -> `is matched`?

Reply:
Thank you. I re-read about this, @beliefer .

In this paragraph, it's a logical continuation of three sentences. (1) If None, (2) If not None, (2-1) If `field` doesn't matched. Since (2) starts to mention that `will contain the matched field`. And, the very next (and final sentence) says, `unmatched` case. I believe the AS-IS is better.
```
a Json object containing the following fields if `field` is None: ...
```
```
If `field` is not None, the Json object will contain the matched field.
```
```
If `field` doesn't match, the Json object `(field -> "")` is returned.
```

Reply:
Got it.